src/components/About.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import tw, { styled } from "twin.macro";
+import { MaxWidthWrapper, TextWrapper } from "./Wrappers";
+import Heading from "./Heading";
+import Link from "./Link";
+
+const About: React.FC = () => {
+  return (
+    <MaxWidthWrapper>
+      <Wrapper>
+        <MainHeading level={3}>About UMA KPI Options</MainHeading>
+        <TextWrapper>
+          <p>
+            UMA will be airdropping a new incentivization mechanism called a
+            “KPI option” to a curated list of addresses in DeFi. The value of
+            these options depends on the TVL of the UMA protocol, thereby giving
+            every recipient an incentive to grow the protocol. (KPI = key
+            performance indicator)
+          </p>
+          <p>
+            Compared to a typical airdrop which can be easily dumped, KPI
```

Comment:
Hm I don't like describing a typical airdrop as one that can be easily "dumped", perhaps its better to describe it as: "KPI Options aim to align airdrop recipient incentives with the long term growth of the protocol"

@UMAprotocol/eng curious what others think

Reply:
I suggest we adjust content in a final PR, since most of it is placeholders for now